python/lsst/meas/base/applyApCorr.py

```diff
@@ -74,32 +77,33 @@ def __init__(self, schema, name):
         self.fluxFlagKey = schema.find(name + "_flag").key
         self.apCorrKey = schema.addField(
             name + "_apCorr",
-            doc = "aperture correction applied to %s" % (name,),
-            type = float,
+            doc="aperture correction applied to %s" % (name,),
+            type=float,
         )
         self.apCorrSigmaKey = schema.addField(
             name + "_apCorrSigma",
-            doc = "aperture correction applied to %s" % (name,),
-            type = float,
+            doc="aperture correction applied to %s" % (name,),
+            type=float,
         )
         self.apCorrFlagKey = schema.addField(
             name + "_flag_apCorr",
-            doc = "set if unable to aperture correct %s" % (name,),
-            type = "Flag",
+            doc="set if unable to aperture correct %s" % (name,),
+            type="Flag",
         )
 
+
 class ApplyApCorrConfig(lsst.pex.config.Config):
     ignoreList = lsst.pex.config.ListField(
-        doc = "flux measurement algorithms in getApCorrNameSet() to ignore;" +
-            " if a name is listed that does not appear in getApCorrNameSet() then a warning is logged",
-        dtype = str,
-        optional = False,
-        default = (),
+        doc="flux measurement algorithms in getApCorrNameSet() to ignore;" +
```

Comment:
Good catch. Also there is a mix of styles for multi-line strings that I standardized.